examples/extensions-zalando/src/main/java/org/zalando/intellij/swagger/examples/extensions/zalando/validator/zally/http/HttpZallyService.java

```diff
@@ -0,0 +1,52 @@
+package org.zalando.intellij.swagger.examples.extensions.zalando.validator.zally.http;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.intellij.openapi.components.ServiceManager;
+import feign.Feign;
+import feign.Logger;
+import feign.codec.Decoder;
+import feign.gson.GsonDecoder;
+import feign.gson.GsonEncoder;
+import org.zalando.intellij.swagger.examples.extensions.zalando.validator.ZallySettings;
+import org.zalando.intellij.swagger.examples.extensions.zalando.validator.zally.ZallyService;
+import org.zalando.intellij.swagger.examples.extensions.zalando.validator.zally.model.LintingRequest;
+import org.zalando.intellij.swagger.examples.extensions.zalando.validator.zally.model.LintingResponse;
+import org.zalando.intellij.swagger.examples.extensions.zalando.validator.zally.model.LintingResponseErrorDecoder;
+
+import java.util.Map;
+
+public class HttpZallyService implements ZallyService {
+
+    public LintingResponse validate(final String spec) {
+        return connect().validate(zallyHeaders(), new LintingRequest(spec));
+    }
+
+    private Map<String, Object> zallyHeaders() {
+        return ImmutableMap.of(
+                "User-Agent", "intellij-swagger-plugin/0.0.1",
```

Comment:
Is it possible to set the version dynamically? Would be nice to know which plugin versions are used.

Reply:
Good idea, fixed in https://github.com/zalando/intellij-swagger/pull/188/commits/13b99f4872a3364b2929caadc01e5b741431e41b